inductiva/structures/scenarios/plate_linear_elastic/geometry_utils.py

```diff
@@ -0,0 +1,614 @@
+"""Utils to create the geometric case."""
+
+from abc import ABC, abstractmethod
+from typing import List, Optional, Tuple
+
+import gmsh
+import json
+import math
+
+
+class Plate(ABC):
+    """Abstract base class for plates."""
+
+    @abstractmethod
+    def perimeter(self):
+        """Abstract method to calculate the perimeter of the plate."""
+        pass
+
+    @abstractmethod
+    def to_dict(self):
+        """Abstract method to convert the plate properties to a dictionary."""
+        pass
+
+    @abstractmethod
+    def to_occ(self):
+        """Abstract method to convert the plate to OpenCASCADE CAD."""
+        pass
+
+    @abstractmethod
+    def get_boundary_ids(self):
+        """Abstract method to get the IDs of the plate boundaries."""
+        pass
+
+    @abstractmethod
+    def get_mesh_metrics(self):
+        """Abstract method to get the metrics for mesh generation in Gmsh."""
+        pass
+
+
+class RectangularPlate(Plate):
+    """Rectangular plate.
+
+    Attributes:
+        plate_type (str): Plate type.
+        length (float): Plate length.
+        width (float): Plate width.
+    """
+
+    def __init__(self, length: float, width: float) -> None:
+        """Initializes a RectangularPlate object."""
+        self.plate_type = "rectangular"
```

Comment:
I think this attribute was unintentionally left here. It isn't needed, so it should be removed.